Security: rename package to project (#28)
packagist/private-packagist-docs: docs/security-monitoring.md

# Security Monitoring

> Private Packagist Security Monitoring searches the dependencies of your projects (private packages with a composer.lock) for known security vulnerabilities.
> Projects are analyzed every time we find a new commit in your project and when a new vulnerability is published in one of the databases.

Reviewer: ```suggestion
Private Packagist Security Monitoring searches the dependencies of your projects for known security vulnerabilities.
Projects, which are Composer packages with a composer.lock file, are analyzed every time you push a new commit to your project and when a new vulnerability is published in one of the databases.
```

The following databases are used to analyze your projects:
* [FriendsOfPHP/security-advisories](https://github.com/FriendsOfPHP/security-advisories)

<div class="row column">
    <div class="callout warning">

>         <p>Security monitoring is available for projects in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>

Reviewer: ```suggestion
        <p>Security monitoring is available for packages in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>
```

    </div>
</div>

### Configuring Monitoring Settings

> The default branch of every project is automatically monitored if a composer.lock file is present.

Reviewer: ```suggestion
The default branch of every project is automatically monitored if a composer.lock file is present.
```

Author: @naderman there are not suggested changes here. Did you miss something?

Reviewer: haha no i meant to undo this, and didn't realize that it's still a "comment" so I cannot just edit the content to be the original to undo the comment :D

Additional branches to be monitored can be selected on the package page.

> Security monitoring can be disabled for individual projects on the package security page or for all projects

Reviewer: ```suggestion
Security monitoring can be disabled for individual project packages on the package security page or for all projects
```
